### PR TITLE
Adds rolling average usage tracking to SS overview

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## Why its good for the game

Makes tracking subsystems which are using more than their fair share of cpu more obvious by having the ability to identify spikes of usage and overall usuage over a set timeframe.

Also cleans up existing UI code that improperly formatted existing figures (assumed say cost was a percentage that would sum to 100% and thus improperly double lowered cost values by a factor of 100)

## Changelog
:cl:
admin: Subsystem Overview now has the ability to track a rolling average of tick by tick subsystem cpu usage.
/:cl:

### CODE REVIEW

**Comment by deathrobotpunch** [comment]
10/10 best pr of the year

File: code/controllers/master.dm [hunk 1]
```diff
@@ -157,6 +159,20 @@ ADMIN_VERB(cmd_controller_view_ui, R_SERVER|R_DEBUG, "Controller Overview", "Vie
 
 	var/list/subsystem_data = list()
 	for(var/datum/controller/subsystem/subsystem as anything in subsystems)
+		var/list/rolling_usage = subsystem.rolling_usage
+		// First, trim er up
+		var/cut_to = 0
+		while(cut_to + 1 <= length(rolling_usage) && text2num(rolling_usage[cut_to + 1]) < (world.time - rolling_usage_length) / world.tick_lag)
```
Comment (MrStonedOne) [inline code/controllers/master.dm:165]: DS2TICK is already a macro and makes the logic easier to understand.

Comment (LemonInTheDark) [reply code/controllers/master.dm]: o7

**Comment by LemonInTheDark** [comment]
Thanks to zypher for writing me a pr description (I am very lazy)

**Pull request merged**